### PR TITLE
New Triage Method

### PR DESCRIPTION
**Pull request by jacobdadams**
This is mostly the same as what was presented in the meeting. 

It sounds like we should iron out the "has been done" signal- checking the checkbox, editing the issue with the initials of the person who did it, etc.

I think there are three goals for the "has been done" signal:

- Unambiguously state the action has been completed.
- Track who completed it.
- Notify everyone involved.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/removing_data_or_service.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+name: Removing Data From Anywhere
```
Comment (steveoh) [inline .github/ISSUE_TEMPLATE/removing_data_or_service.md:2]: i think we shoudl replace the existing issue, will you rename this to, `deprecate-sgid-dataset-or-web-application.md`

File: README.md [hunk 2]
```diff
@@ -4,8 +4,10 @@ Track deprecation tasks
 
 ## How to use
 
-1. When an application, dataset, or both are requesting deprecation [create a new issue](https://github.com/agrc/cemetery/issues/new/choose)
-1. Choose the most applicable template from the buttons
-1. place an `[x]` in the relevant boxes and submit the issue
-1. tag anyone who might be a stakeholder in the deprecation
-1. follow through with the all of the items before closing the issue
+1. When an application, dataset, or both needs to be deleted or moved to a different long-term storage, [create a new issue](https://github.com/agrc/cemetery/issues/new/choose)
+1. Choose the `removing_data_or_service` template from the buttons
```
Comment (steveoh) [inline README.md:8]:
```suggestion
1. Choose the `Deprecate SGID Dataset or Web Application` template from the buttons
```

File: README.md [hunk 3]
```diff
@@ -4,8 +4,10 @@ Track deprecation tasks
...
+1. Tag the three triagers (one from each team) in the triage section
```
Comment (steveoh) [inline README.md:9]: I think the dev team will handle this differently so we can leave that area blank and we will fill it in. do we want to edit the template to add the data team and casastre team triagers so the submitter doesn't have to know who they are?

File: README.md [hunk 4]
```diff
@@ -4,8 +4,10 @@ Track deprecation tasks
...
+1. The triagers will choose which items need to be handled and tag the appropriate people to handle them
+   - Items that don't need to be handled should be struck through (wrap the line with `~`'s) instead of being deleted
+1. The people assigned to different issues will check the boxes as they are completed
```
Comment (steveoh) [inline README.md:12]:
```suggestion
1. The people assigned to different issues will check the boxes as they are completed and comment on the issue to notify the rest of the group.
```

File: .github/ISSUE_TEMPLATE/removing_data_or_service.md (hunk 1 above)
Comment (jacobdadams) [reply .github/ISSUE_TEMPLATE/removing_data_or_service.md:2]: Yeah, that makes sense, I'll switch it over.

File: README.md (hunk 3 above)
Comment (jacobdadams) [reply README.md:9]: Yeah, that was what I was thinking, I just didn't/don't know who they'll be.

**Pull request merged**